docs/src/examples/components/TextArea/Usage/TextAreaMaxLengthExample.shorthand.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react'
+import { TextArea } from '@stardust-ui/react'
+
+const TextAreaMaxLengthExample = () => (
+  <TextArea placeholder={`No more than 5 letters can be here...`} maxLength={5} />
```

Comment:
```suggestion
  <TextArea placeholder='No more than 5 letters can be here...' maxLength={5} />
```

_nit_, should be handled in future by ESLint